test/js/components/CommitId-spec.jsx

```diff
@@ -24,4 +24,10 @@ describe("CommitId", () => {
         assert.equal(wrapper.text().length, 7);
         assert.equal(wrapper.text(), '676b757');
     });
+
+    it("renders link when url prop is passed along", () => {
+        const wrapper = shallow(<CommitId commitId="123" url="link" />);
+        assert.isTrue(wrapper.is('a'));
+        assert.equal(wrapper.html(), '<a href="link" target="_blank" title="Opens commit in a new window"><code class="hash">123</code></a>');
```

Comment:
You could make a change here too but I don't think it's absolutely required. Generally with Enzyme I use one of two different approaches:
- Test for presence of nodes with `find()` or against node values using `prop(name)`, `text()` and similar. This seems to be the most popular thing to do but can be tedious if you have a lot of nested nodes - especially nested components - since you need to call `shallow()` on each comp. (Although at that point, it's more of an integration test than a pure unit test).
- Test with `html()` and `indexOf` to check that HTML contains certain string fragments you might expect. This tends to be very fast and easy to work with.

I would be cautious about using `html()` and an exact string comparison just because it will break in the event of any change made to the component.